src/openassetio-python/package/openassetio/test/manager/apiComplianceSuite.py

```diff
@@ -758,23 +758,23 @@ class Test_getWithRelationshipPaged(PagedTest):
     """
 
     def test_when_relation_unknown_then_no_pages_returned(self):
-        a_ref = self.requireEntityReferenceFixture("a_reference", skipTestIfMissing=False)
```

Comment:
[question][plan] Be great if we had a way to say "you implemented one of the 4 relationships methods, make sure you go implement the others".

Reply:
Closing as this would be fixed by the proposed changes in the other comment about 🌧️